src/router/storeSync/LayerParamConfig.class.js

```diff
@@ -13,7 +13,7 @@ import log from '@/utils/logging'
  * layer and its state (visibility, opacity, etc...)
  *
  * @param {AbstractLayer} layer
- * @param {GeoAdminLayer[]} defaultLayerConfig
```

Comment:
that might be the linter going nuts, or that we have different JSDocs parser you and I, but this is unwanted I assume 😉

Reply:
Nope based on jsdoc https://jsdoc.app/tags-param.html optional parameter are defined like this
```
/**
 * @param {string} [somebody] - Somebody's name.
 */
function sayHello(somebody) {
    if (!somebody) {
        somebody = 'John Doe';
    }
    alert('Hello ' + somebody);
}
```